lib/resources/oracledb_session.rb

```diff
@@ -41,7 +41,7 @@ def initialize(opts = {})
       @db_role = opts[:as_db_role]
 
       # we prefer sqlci although it is way slower than sqlplus, but it understands csv properly
-      @sqlcl_bin = 'sql' unless opts.key?(:sqlplus_bin) # don't use it if user specified sqlplus_bin option
+      @sqlcl_bin = opts[:sqlcl_bin] || 'sql' unless opts.key?(:sqlplus_bin) # don't use it if user specified sqlplus_bin option
```

Comment:
This appears to introduce a new resource parameter, `sqlcl_bin` but does not document that in the documentation.

Reply:
`sqlcl_bin` was an existing resource parameter that is documented but was previously unused in the code.
https://github.com/inspec/inspec/blob/8cdd7cfe0309528e2c866e1c311ee854e012e94a/docs/resources/oracledb_session.md.erb#L34

Reply:
Ahh, thanks!